source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/InputDateRendererTests.swift

```diff
@@ -137,6 +137,32 @@ class InputDateRendererTest: XCTestCase {
         XCTAssertEqual(inputDateField.textField.accessibilityValue(), "10-Feb-2000")
     }
     
+    func testInvalidDateRejected() {
+        let minVal = "2023-04-05"
+        let maxVal = "2023-04-25"
+        
+        //date less than min date
+        var val = "2023-04-02"
+        inputDate = .make(value: val, max: maxVal, min: minVal)
+        var inputDateField = renderDateInput()
+        
+        XCTAssertFalse(inputDateField.isValid)
```

Comment:
@mukuagar shouldn't you also test if your delegate is called or not (because tomorrow anyone can remove that delegate and that test will pass)